main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
Actually I would prefer to maintain DB schema version manually. Using git revision seems like an overkill. Unless your goal is to have random looking versions IDs instead of numbers. But then, it's rather complicated to figure our how these versions are ordered. On top of this, I don't see how we're going to organize the process of assigning IDs in such case. You don't know the ID. So even if we take n-1 commit, if you rebase, the hash would change, hence you'd have to update schema version.

Besides, version information in `version.go` is only set when building in travis. I don't think that anyone sets it when building locally. Just another source of confusion.